farsetlabs.json

```diff
@@ -0,0 +1,38 @@
+{
+    "api": "0.13",
+    "botname": "FarBot",
+    "spacefed": {
+        "spacenet": false,
+        "spacesaml": false,
+        "spacephone": false
+    },
+    "contact": {
+        "email": "admin@farsetlabs.org.uk",
+        "github":"https://github.com/FarsetLabs",
+        "twitter": "@farsetlabs",
```

Comment:
Everything else in the `contact` sections is a URL should his also be a URL?

Reply:
I htink ti will automatically format the twitter name as a URL/link, it just doesn't handle the others as well